firmware/src/sys/board.hpp

```diff
@@ -58,6 +58,9 @@ void resetWatchdog();
 
 void setStatusLed(bool state);
 void setCanLed(bool state);
+void DutycycleCounterPP();
+void DutycycleCounterMM();
+signed DutycycleCounterRead();
```

Comment:
What is the purpose of this variable? The fact that it needs to be accessible from both hints that there's something wrong with the design.